### PR TITLE
fix: TVL api

### PR DESCRIPTION
**Pull request by witherblock**

### CODE REVIEW

File: helpers/v2/getVedpxTvl.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { utils as ethersUtils } from 'ethers'
+
+import getProvider from '../getProvider'
+import getPrice from '../getPrice'
+
+export default async () => {
+    const provider = getProvider(42161)
+
+    const dpxContract = ERC20__factory.connect(
+        '0x6c2c06790b3e3e3c38e12ee22f8183b37a13ee55',
```
Comment (garyunwin42) [inline helpers/v2/getVedpxTvl.js:11]: why dont we grab the address from the SDK?

File: api/v2/tvl/[product].js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+export default async (req, res) => {
+    try {
+        const product = req.query.product
+
+        let tvl = 0
+
+        switch (product) {
+            case 'ssov': {
+                const _SSOVS = SSOVS.filter((ssov) => !ssov.retired)
```
Comment (garyunwin42) [inline api/v2/tvl/[product].js:17]: redundant coz we already check https://github.com/dopex-io/dopex-api/blob/master/helpers/v2/getSsovTvl.js#L13. but it's fine if we keep it

File: helpers/v2/getVedpxTvl.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        provider
+    )
+
+    let tvl = '0'
```
Comment (garyunwin42) [inline helpers/v2/getVedpxTvl.js:15]: don't string this? if error, we can return 0

File: api/v2/tvl/[product].js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                tvl = farmTvls.reduce((acc, item) => {
+                    return acc + item.toNumber()
+                }, 0)
+
+                tvl = String(tvl)
+                break
+            }
+
+            case 'vedpx': {
+                tvl = await getVedpxTvl()
```
Comment (garyunwin42) [inline api/v2/tvl/[product].js:65]: string this?

File: helpers/v2/getVedpxTvl.js (hunk 3 above)
Comment (witherblock) [reply helpers/v2/getVedpxTvl.js]: Fixed

File: api/v2/tvl/[product].js (hunk 4 above)
Comment (witherblock) [reply api/v2/tvl/[product].js:65]: Fixed

File: helpers/v2/getVedpxTvl.js (hunk 1 above)
Comment (witherblock) [reply helpers/v2/getVedpxTvl.js]: Fixed, but in general we are starting to move away from using addresses from the sdk

**Review by garyunwin42 — APPROVED** [review]

**Pull request merged**